tool/rule.dart

```diff
@@ -128,13 +128,20 @@ const _details = r'''
 ''';
 
 class $className extends LintRule {
+  static const LintCode code = LintCode(
```

Comment:
We might want to add documentation somewhere for how to add more than one code, but this looks good.

Reply:
In fact, we might inline some docs here in general.  (Like the Flutter templates.)  Maybe a link to your diagnostic writing doc?

/fyi @MaryaBelanger